### ISSUE TITLE
[java] IdenticalCatchBranches false positive

### ISSUE THREAD

**Issue by adangel** [pmd/pmd#1480]
**Affects PMD Version:** 6.9.0

**Rule:** [IdenticalCatchBranches](https://pmd.github.io/pmd-6.9.0/pmd_rules_java_codestyle.html#identicalcatchbranches)

**Description:** False positive if only the return statement expression is different (true vs. false in the example below).

**Code Sample demonstrating the issue:**

```
public class Foo {
    public boolean classNameExists(String fullyQualifiedClassName) {
        try {
            pmdClassLoader.loadClass(fullyQualifiedClassName);
            return true; // Class found
        } catch (ClassNotFoundException e) {
            return false;
        } catch (LinkageError e2) {
            // Class exists, but may be invalid (see https://github.com/pmd/pmd/issues/1131)
            return true;
        }
    }
}
```

**Running PMD through:** Maven
